src/components/IOUConfirmationList.js

```diff
@@ -187,10 +194,18 @@ class IOUConfirmationList extends Component {
 
             const formattedMyPersonalDetails = OptionsListUtils.getIOUConfirmationOptionsFromMyPersonalDetail(
                 this.props.currentUserPersonalDetails,
-                this.props.numberFormat(IOUUtils.calculateAmount(selectedParticipants, this.props.iouAmount, true) / 100, {
-                    style: 'currency',
-                    currency: this.props.iou.selectedCurrencyCode,
-                }),
+                this.props.numberFormat(
+                    IOUUtils.calculateAmount(
+                        selectedParticipants,
+                        this.props.iouAmount,
+                        this.props.iou.selectedCurrencyCode,
+                        true,
+                    ) / IOUUtils.getCurrencyUnits(this.props.iou.selectedCurrencyCode),
+                    {
+                        style: 'currency',
+                        currency: this.props.iou.selectedCurrencyCode,
+                    },
+                ),
```

Comment:
Can you put this in a variable with a name, I think it will help with making the code more clear.

Reply:
or maybe just the value returned by `IOUUtils.calculateAmount(...)`